controllers/childGuidance.js

```diff
@@ -0,0 +1,133 @@
+import childModel from '../models/childModel.js';
+
+const createGuidance = async (req, res) => {
+  try {
+    const data = req.body;
+    const check = await childModel.findOne({ guideId: data.guideId });
```

Comment:
@robsdagreat Good job, please work on the following changes:

- why in the body of the request are you sending the guideId?

Reply:
I was trying to add a unique id that is different with the default monogDB id but I'm going  to change it